src/Value/Tracking/ClickMetadata.php

```
@@ -0,0 +1,72 @@
+    }
+
+    public static function fromArray(array $raw): self
+    {
+        return new self(
+            isset($raw['market']) ?
+                Market::from($raw['market']) :
+                null,
+            isset($raw['ipAddress']) ?
+                $raw['ipAddress'] :
+                null,
+            isset($data['userAgent'])
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (undefined)

(UnusedLocalVariable)